MAKE/Makefile.minihpc

```diff
@@ -0,0 +1,84 @@
+#Kostis Papadakis
+#======== Vectorization ==========
+#Set vector backend type for vlasov solvers, sets precision and length.
+#NOTE this has to have the same precision as the distribution function define (DISTRIBUTION_FP_PRECISION)
+#Options:
+# AVX:	    VEC4D_AGNER, VEC4F_AGNER, VEC8F_AGNER
+# AVX512:   VEC8D_AGNER, VEC16F_AGNER
+# Fallback: VEC4D_FALLBACK, VEC4F_FALLBACK, VEC8F_FALLBACK
+
+ifeq ($(DISTRIBUTION_FP_PRECISION),SPF)
+#Single-precision
+	VECTORCLASS = VEC4F_FALLBACK
+else
+#Double-precision
+	VECTORCLASS = VEC4D_FALLBACK
+endif
+
+
+#======== Allocator =========
+#Use TBB malloc
+
+CMP = mpic++
+LNK = mpic++
+
+PAPI_FLAG =
```

Comment:
PAPI_FLAG does not exist (though this does no harm, it can be confusing if you want to turn it on at some point). And if you set PAPI active with CXX_FLAGS, it needs to be lower down (see the makefile cleanup PR to urs' repo)